components/ai_chat/resources/page/components/main/index.tsx

```diff
@@ -250,7 +250,7 @@ function Main() {
                     />
                   }
                 </div>
-
+                <div className={styles.bottomContent}>
```

Comment:
so error's and stuff still display properly with this? Sick!

one last nit - I think the children of this should be indented.

Reply:
Actually, thinking about this, maybe we don't need it to wrap the content at all? Maybe we could put a negative `margin-bottom` on the `div ref={scrollAnchor}` above?